### PR TITLE
stackops

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: create-subscription-acc/request-for-techbiz-account.md [hunk 1]
```diff
@@ -109,10 +109,10 @@ Have the following information ready before proceeding:
 
 
 
-## Step 3: Select SGTS Products for subscription
+## Step 3: Select SGTS Services for subscription
 
 
-In **Select product(s)**, choose the required **SGTS Service(s)** and click **Next**.
+In **Select service(s)**, choose the required **SGTS Service(s)** and click **Next**.
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:115]: typo : Service(s)

File: create-subscription-acc/request-for-techbiz-account.md [hunk 2]
```diff
@@ -130,11 +130,11 @@ In **Select product(s)**, choose the required **SGTS Service(s)** and click **Ne
...
 | **Field Name** | **Description** |
 | :------------------------------------------ | ------------------ |
-| **Select subscription period** | Select the start date of the subscription. </br></br>This is the date when your agency intends to start using the subscribed SGTS Products. |
+| **Select subscription period** | Select the start date of the subscription. </br></br>This is the date when your agency intends to start using the subscribed SGTS Services. |
 | **Product agreement** | Click to view the terms and conditions of the product. |
-| **Select a plan** | Select the required subscription plan for a product.</br></br>The displayed subscription plans contain details of the chargeable items in a product and their charges. |
+| **Select a plan** | Select the required subscription plan for a product.</br></br>The displayed subscription plans contain details of the chargeable items in a service and their charges. |
 
-4. If you are subscribing to more than one SGTS product, repeat **steps 2-3** and click **Next**.
+4. If you are subscribing to more than one SGTS services, repeat **steps 2-3** and click **Next**.
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:137]: typo: Services

File: create-techbiz-system.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 A TechBiz system allows agencies to share usage of their [Singapore Government Tech Stack (SGTS)](https://www.developer.tech.gov.sg/singapore-government-tech-stack/overview/index.html) product subscriptions which can be configured and managed individually. A system is connected to a TechBiz account. An account can have multiple systems. Refer to the [FAQ](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/faq) for more information.
 
-Agencies need to add and configure TechBiz systems to manage the subscribed SGTS products.
+Agencies need to add and configure TechBiz systems to manage the subscribed SGTS services.
```
Comment (denisecollard) [inline create-techbiz-system.md:5]: Typo: Services

File: create-techbiz-system.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ This sections guides how subscription admins can create a TechBiz system using a
 | **Field Name** | **Description** |
 | --- | --- |
 | **System name** | Enter the system name.</br><br/> **Note:** System name can have a minimum of 1 character and maximum of 50 characters. |
-| **TechBiz system ID** | A unique TechBiz system ID is automatically generated and cannot be edited. The TechBiz system ID is common across all subscribed SGTS products under this account. |
+| **TechBiz system ID** | A unique TechBiz system ID is automatically generated and cannot be edited. The TechBiz system ID is common across all subscribed SGTS services under this account. |
```
Comment (denisecollard) [inline create-techbiz-system.md:34]: Typo: Services

File: add-and-configure-products.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
-# Add and configure products
+# Add and configure services
 
-Agencies need to add the subscribed SGTS Products required for the system(s) and configure the Products for each TechBiz system.
+Agencies need to add the subscribed SGTS Services required for the system(s) and configure the Services for each TechBiz system.
 
-**To add and configure subscribed products**
+**To add and configure subscribed services**
```
Comment (denisecollard) [inline add-and-configure-products.md:5]: Typo: Services

File: create-techbiz-system.md [hunk 6]
```diff
@@ -52,4 +52,4 @@ The system record appears on the **Systems** page with status as **Pending Confi
 
 <kbd><img src="images/pending-config.png" alt="drawing" width="100%"/></kbd>
 
-Proceed to [Add and configure products](add-and-configure-products.md)  complete the configuration.
+Proceed to [Add and configure services](add-and-configure-products.md)  complete the configuration.
```
Comment (denisecollard) [inline create-techbiz-system.md:55]: Typo: Services

File: home.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ Welcome to the TechBiz documentation!
 | [Manage draft request](create-subscription-acc/manage-draft-request.md) </br></br> Manage your draft subscription account. | [Resubmit account request](create-subscription-acc/resubmit-techbiz-account-application.md)</br></br> Resubmit your account request.  |
 |  [Manage subscriptions](manage-subscriptions.md) </br></br> Get information on managing your subscriptions. |  [Manage subscription account](manage-techbiz-account.md)</br></br> Get information on managing your subscription account. |
 | [Create system](create-techbiz-system.md) </br></br> Create a system on TechBiz. | [Manage system](manage-techbiz-system.md)</br></br> Manage your existing systems. |
-| [Add and configure products](add-and-configure-products.md) </br></br> Add and configure products to your system. | [User management](manage-user-access-subscribed-sgts-products.md) </br></br> Use TechBiz to add or remove users. |
+| [Add and configure services](add-and-configure-products.md) </br></br> Add and configure services to your system. | [User management](manage-user-access-subscribed-sgts-products.md) </br></br> Use TechBiz to add or remove users. |
```
Comment (denisecollard) [inline home.md:12]: Typo: Services

File: manage-announcements.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Manage announcements
 
-This section guides how administrators can create announcements and manage them on [TechBiz portal.](https://portal.techbiz.suite.gov.sg/) Announcements allow SGTS product owners to broadcast messages on TechBiz portal. It appears as a pop up after a user has logged in to TechBiz portal.
+This section guides how administrators can create announcements and manage them on [TechBiz portal](https://portal.techbiz.suite.gov.sg/). Announcements allow SGTS product owners to broadcast messages on TechBiz portal. It appears as a pop up after a user has logged in to TechBiz portal.
```
Comment (denisecollard) [inline manage-announcements.md:3]: Typo: SGTS Service owners

File: manage-user-access-subscribed-sgts-products.md [hunk 9]
```diff
@@ -33,7 +33,7 @@ Subscription and technical admins can control user access to the SGTS Products t
 
 <kbd><img src="images/user-man.png" alt="drawing" width="100%"/></kbd>
 
-5. Select the required product from the **Products** dropdown list to select the SGTS product tools to manage.
+5. Select the required product from the **Services** dropdown list to select the SGTS service tools to manage.
```
Comment (denisecollard) [inline manage-user-access-subscribed-sgts-products.md:36]: Typo: Service

File: manage-user-access-subscribed-sgts-products.md [hunk 10]
```diff
@@ -86,7 +86,7 @@ Subscription and technical admins can control user access to the SGTS Products t
 2. Select **APPROVED** from the **Status** dropdown list to view all the approved accounts.
 3. Select the required account.
 4. Click **USER MANAGEMENT** tab.
-5. Select the required product from the **Products** dropdown list to select the SGTS product tools to manage.
+5. Select the required product from the **Services** dropdown list to select the SGTS product tools to manage.
```
Comment (denisecollard) [inline manage-user-access-subscribed-sgts-products.md:89]: Typo: SGTS Services tools

**Review by denisecollard — CHANGES_REQUESTED** [review]
Need to amend with upper case S for "Services"

File: create-subscription-acc/request-for-techbiz-account.md [hunk 11]
```diff
@@ -134,7 +134,7 @@ In **Select service(s)**, choose the required **SGTS Service(s)** and click **Ne
 | **Product agreement** | Click to view the terms and conditions of the product. |
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:134]: Typo: Service

File: create-subscription-acc/request-for-techbiz-account.md [hunk 12]
```diff
@@ -134,7 +134,7 @@ In **Select service(s)**, choose the required **SGTS Service(s)** and click **Ne
...
 | **Select a plan** | Select the required subscription plan for a product.</br></br>The displayed subscription plans contain details of the chargeable items in a service and their charges. |
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:135]: Typo: Service

**Review by denisecollard — CHANGES_REQUESTED** [review]
Noticed a couple of "product" term not changed to "Service"

File: admin-roles-and-permissions.md [hunk 13]
```diff
@@ -12,8 +12,8 @@ Following roles and permissions are available for agencies to administer their s
 
 - Create TechBiz subscription accounts and systems
 - Manage users
-- Manage product subscriptions and plans
-- Provide product configuration details
+- Manage Service subscriptions and plans
+- Provide Serviceconfiguration details
```
Comment (denisecollard) [inline admin-roles-and-permissions.md:16]: Spacing in between: Service configuration

File: create-subscription-acc/request-for-techbiz-account.md [hunk 14]
```diff
@@ -109,16 +109,16 @@ Have the following information ready before proceeding:
 
 
 
-## Step 3: Select SGTS Products for subscription
+## Step 3: Select SGTS Services for subscription
 
 
-In **Select product(s)**, choose the required **SGTS Service(s)** and click **Next**.
+In **Select service(s)**, choose the required **SGTS Service(s)** and click **Next**.
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:115]: Capital S: Service(s)

File: create-subscription-acc/request-for-techbiz-account.md [hunk 15]
```diff
@@ -130,11 +130,11 @@ In **Select product(s)**, choose the required **SGTS Service(s)** and click **Ne
 
 | **Field Name** | **Description** |
 | :------------------------------------------ | ------------------ |
-| **Select subscription period** | Select the start date of the subscription. </br></br>This is the date when your agency intends to start using the subscribed SGTS Products. |
-| **Product agreement** | Click to view the terms and conditions of the product. |
-| **Select a plan** | Select the required subscription plan for a product.</br></br>The displayed subscription plans contain details of the chargeable items in a product and their charges. |
+| **Select subscription period** | Select the start date of the subscription. </br></br>This is the date when your agency intends to start using the subscribed SGTS Services. |
+| **Service agreement** | Click to view the terms and conditions of the Service. |
+| **Select a plan** | Select the required subscription plan for a service.</br></br>The displayed subscription plans contain details of the chargeable items in a service and their charges. |
```
Comment (denisecollard) [inline create-subscription-acc/request-for-techbiz-account.md:135]: Capital S : a Service

File: glossary.md [hunk 16]
```diff
@@ -13,11 +13,11 @@ Government standard image build (GSIB) is a standard operating environment (SOE)
 A non-secured email (SE) GSIB device.
 
 **Account**
-A TechBiz account allows agencies to subscribe to SGTS Products, add systems to the account, manage subscription plans and view billing details. An agency can have one account for multiple systems or an account for each system.
+A TechBiz account allows agencies to subscribe to SGTS Services, add systems to the account, manage subscription plans and view billing details. An agency can have one account for multiple systems or an account for each system.
 
 **System**
 
-A TechBiz system allows agencies to centrally manage and view their SGTS product subscription plans, billing details, and resource usage within a system. An agency can have multiple systems.
+A TechBiz system allows agencies to centrally manage and view their SGTS service subscription plans, billing details, and resource usage within a system. An agency can have multiple systems.
```
Comment (denisecollard) [inline glossary.md:20]: Capital S: SGTS Service

File: manage-user-access-subscribed-sgts-products.md [hunk 17]
```diff
@@ -33,7 +33,7 @@ Subscription and technical admins can control user access to the SGTS Products t
 
 <kbd><img src="images/user-man.png" alt="drawing" width="100%"/></kbd>
 
-5. Select the required product from the **Products** dropdown list to select the SGTS product tools to manage.
+5. Select the required Service from the **Services** dropdown list to select the SGTS service tools to manage.
```
Comment (denisecollard) [inline manage-user-access-subscribed-sgts-products.md:36]: Capital S: SGTS Service

**Review by denisecollard — CHANGES_REQUESTED** [review]
To amend "Services" with capital "S" for consistency

File: manage-user-access-subscribed-sgts-products.md (hunk 17 above)
Comment (kpslp) [reply manage-user-access-subscribed-sgts-products.md]: fixed

File: glossary.md (hunk 16 above)
Comment (kpslp) [reply glossary.md]: fixed

File: create-subscription-acc/request-for-techbiz-account.md (hunk 15 above)
Comment (kpslp) [reply create-subscription-acc/request-for-techbiz-account.md]: fixed

File: create-subscription-acc/request-for-techbiz-account.md (hunk 14 above)
Comment (kpslp) [reply create-subscription-acc/request-for-techbiz-account.md]: fixed

File: admin-roles-and-permissions.md (hunk 13 above)
Comment (kpslp) [reply admin-roles-and-permissions.md]: fixed

**Review by denisecollard — APPROVED** [review]

**Pull request merged**